CDCgov/prime-reportstream
# 11050: handle cancelled OBX segments

This PR updates the translations to support cancelled OBX segments

Test Steps:

You'll need to set the `DEV_ELIMS` in organizations.yml to active

For each of the new ELIM tests files
1. Run them through the UP
2. Verify that the outputted HL7 file matches the input exactly with the following exceptions
  - There are still instances where ELIMS is not sending a full timestamp
  - Some the messages, ELIMS is not including MSH.18 and it gets added 

## Changes
- Add mapping for cancelled (X status) and not asked (N status) for OBX segments
- Fix the FHIR -> HL7 mapping to set OBX.2 to `ST` only when the reason for no value is `unknown`

## Checklist

### Testing

### Process

## Linked Issues
- #11050 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
